.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,15 +3,17 @@
   WIP: work in progress
   🐛    Fix a bug.
   ✨    Introduce new features.
-  ♻️     Refactor code.
+  🎨    Enhance existing feature.
+  ♻️    Refactor code.
   🚑️    Critical hotfix.
-  ⚗️     Perform experiments.
-  ⬆️     Upgrade dependencies.
+  ⚗️    Perform experiments.
+  ⬆️    Upgrade dependencies.
   📝    Add or update documentation.
   🔨    Add or update development scripts.
   🔒️    Fix security issues.
-  ⚠️     Changes in devops configuration
-  🗃️    Migration of database
+  ⚠️    Changes in ops configuration etc. required before deploying. Please link associated issue or PR.
```

Comment:
What do you mean by "Please link associated issue or PR"? are you talking about osparc-ops-environment? if yes please add the link to the repo. and also explain what is expected thanks.